examples/README.md

```diff
@@ -5,3 +5,4 @@ A collection of small apps or pages using MDC React components.
 Example | Link
 --- | ---
 Roses | [./roses](./roses)
+Catalog | [./catalog](./catalog)
```

Comment:
> * I need to download/clone the repo then run `npm start` to see the catalog, but since the package.json deps needs to be bumped manually on each release (instead of referencing the packages in their respective paths since this is a monorepo) I might be looking at other components than the ones in the repo I just downloaded/cloned.
>   
>   * There would be more use for developers on this repo if it referenced the packages in this repo, so they can see how their changes look (I'd love that!)

It's a good idea!
But there is one worried thing.
If we referenced package.json of root directory, this example isn't always stable.
For example, now `Roses` example used version "^0.5.x".
Maybe this example isn't working with latest version.

Reply:
>   * If I run `npm build`, what am I expected to do with the output?
>     
>     * Could this be put online somehow? I guess this is a question to the maintainers and is a much bigger endeavor.

I exported `index.html`/`main.[hash].js`/`[some static files]` in `dist` directory.
Of course you can serve this files in web server.
Just copy this directory and serve html file.
But now I'm not configured about routing system.
I'll make a next.js example as soon as possible.
That example makes more clear about deploy!

Reply:
Yeah agreed - it could be out of date like Roses. We haven't had the resources to focus on the examples. I think having each example as a smaller sub project makes sense since it requires minimal maintenance down the road.

Reply:
I think publishing it is fairly easy, but it requires maintenance that I don't think the team can take on. I think this is actually a common pattern (having to clone/download the repo and running `npm run start`).